### DISCUSSION TITLE
[Question] How AppFlowy is using Postgres ?

### DISCUSSION THREAD

**Discussion by khanakia** [AppFlowy-IO/AppFlowy#253] [Technical help wanted] [answered]
Hello,

I can see here https://github.com/AppFlowy-IO/appflowy/blob/main/backend/src/config/configuration.rs appflowy seems to be using Postgres but when I open the app I do not see it installs any Postgres server.

So I would really appreciate if you can explain how appflowy is using Postgres under the hood without installing the psql server at all.

**Comment by appflowy** [comment, marked as answer]
Hi @khanakia, at this moment, v 0.0.2 is not using Postgres, as it only supports offline. We are developing an online version (v 0.0.4), which will enable the frontend (desktop application) to communicate with the backend that uses Postgres.

**Comment by khanakia** [comment]
@appflowy so are you using Sqlite as database ?

**Comment by appflowy** [comment]
@khanakia Yes. You can checkout the frontend/rust-lib/flowy-database crate.
